src/js/edu-benefits/5490/config/form.js

```diff
@@ -30,8 +33,8 @@ const {
 } = fullSchema5490.properties;
 
 const {
-  secondaryContact,
-  nonMilitaryJobs
+  nonMilitaryJobs,
+  secondaryContact
```

Comment:
Just trying to keep things alphabetized. Not a huge deal, I know, but it makes me feel better about my life.